### DISCUSSION TITLE
Connection to OpenSea using multisig

### DISCUSSION THREAD

**Discussion by davidseegers** [WalletConnect/walletconnect-monorepo#2383] [Developer Support]
Hi all!

I'm trying to connect to Opensea with our multisig wallet. After connecting with WalletConnect, I need to sign a gasless transaction to accept the Opensea terms of service. We've signed this transaction with the multisig wallets twice now and pushed the transaction through, but the Opensea interface keeps asking us to accept the terms of service. So basically it is impossible to use. Any solutions?

**Comment by crypblizz8** [comment]
This may be an OpenSea issue but would you mind showing a screenrecord or instructional reproduction steps:

My assumption:
- Open opensea
- Try to sign from a Safe wallet
- Does not connect/sign?

**Reply by davidseegers** [reply]
> This may be an OpenSea issue but would you mind showing a screenrecord or instructional reproduction steps:
> 
> My assumption:
> 
> * Open opensea
> * Try to sign from a Safe wallet
> * Does not connect/sign?

1. Connect on OpenSea, copy wallet connect link
2. Go back to wallet connect app page, paste and enter the link
3. Go back to OpenSea, now 'logged in' but shown a pop up to accept terms of service
4. Press accept, needs to be signed with a gasless transaction
5. Go back to wallet connect app page, the transaction shows up
6. I accept it and forward it to the multisig
7. Other signers confirm the transaction to reach multisig consensus
8. Transaction is pushed through
9. Go back to OpenSea, nothing happens. Still standing on the terms of service pop up awaiting signature.

**Reply by finessevanes** [reply]
@davidseegers are you having the same issue when using Safe?
